lib/src/builder/codegen/accessors_generator.dart

```diff
@@ -149,6 +150,8 @@ abstract class TypedMapAccessorsGenerator extends BoilerplateDeclarationGenerato
 
     StringBuffer output = StringBuffer();
 
+    final requiredPropChecks = [];
```

Comment:
This is missing a type:
```suggestion
    final requiredPropChecks = <String>[];
```